### PR TITLE
[status] make jsoniter 100% stdlib compatible

### PR DESCRIPTION
**Pull request by truthbk**
### What does this PR do?

Makes `jsoniter` fully compatible with the stdlib `encoding/json` lib to avoid breaking the templating.

### Motivation

The default `jsoniter` config would post `uint64` values where `encoding/json` returned `float64`. We make it fully compatible to avoid breaking the rendered stats.

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -1666,10 +1666,12 @@
     "k8s.io/client-go/discovery",
     "k8s.io/client-go/dynamic",
     "k8s.io/client-go/informers",
+    "k8s.io/client-go/informers/autoscaling/v2beta1",
```
Comment (arbll) [inline Gopkg.lock]: Should this be included ?

**Review by arbll — DISMISSED** [review]

**Comment by remicalixte** [comment]
It's interesting because we mainly use the numbers as int, so with the standard libs we had to do some conversion in the templates. We could remove this flag in the future and simplify some code

File: Gopkg.lock (hunk 1 above)
Comment (devonboyer) [reply Gopkg.lock]: These are packages we are using for features in the DCA, so yes I think they do need to be in here.

**Comment by truthbk** [comment]
Had already beeen approved, simply had to fix a conflict. Merging.

**Pull request merged**